CHANGELOG.txt

```diff
@@ -1,3 +1,6 @@
+* 4.1.1
```

Comment:
Out of curiosity, what's the versioning policy we're following?

Reply:
Should be Semantic Versioning. I changed the version as I'm not fixing a bug.